app/src/webkit/java/org/mozilla/focus/web/FocusWebViewClient.java

```diff
@@ -43,6 +43,49 @@ public void setCallback(IWebView.Callback callback) {
         this.callback = callback;
     }
 
+    private static final String CLEAR_VISITED_CSS =
+            "nSheets = document.styleSheets.length;" +
+            "for (s=0; s < nSheets; s++) {" +
+            "  stylesheet = document.styleSheets[s];" +
+            " nRules = stylesheet.cssRules ? stylesheet.cssRules.length : 0;" +
+            // rules need to be removed by index. That modifies the whole list - it's easiest
+            // to therefore process the list from the back, so that we don't need to care about
+            // indexes changing after deletion (all indexes before the removed item are unchanged,
+            // so by moving towards the start we'll always process all previously unprocessed items -
+            // moving in the other direction we'd need to remember to process a given index
+            // again which is more complicated).
+            "  for (i = nRules - 1; i >= 0; i--) {" +
+            "    cssRule = stylesheet.cssRules[i];" +
+            // Depending on style type, there might be no selector
+            "    if (cssRule.selectorText && cssRule.selectorText.includes(':visited')) {" +
+            "      stylesheet.deleteRule(i);" +
+            "    }" +
+            "  }" +
+            "}";
```

Comment:
When executing this in the global scope of the website we should try to avoid potentially breaking the site's JS.

How about wrapping this in:

```
(function() {
 ...
})();
```

This will generate an anonymous function and execute it immediately. In addition to that we should use `let` to avoid overriding global variables.